EventHandlers/PrebidMobileMAXAdapters/Sources/PrebidMAXMediationAdapter.swift

```diff
@@ -42,7 +42,6 @@ public class PrebidMAXMediationAdapter: ALMediationAdapter {
     public weak var nativeDelegate: MANativeAdAdapterDelegate?
     
     public override func initialize(with parameters: MAAdapterInitializationParameters, completionHandler: @escaping (MAAdapterInitializationStatus, String?) -> Void) {
-        super.initialize(with: parameters, completionHandler: completionHandler)
         Targeting.shared.subjectToCOPPA = ALPrivacySettings.isAgeRestrictedUser()
```

Comment:
Hi @kubamatula ! Agree, it make sense to signal about initialization completion once all needed steps are done. 

Also, I would like to note that we will review the Prebid SDK initialization flow in case of integration via AdMob/MAX adapters in a way recommended in the development guides. To follow the adapters' implementation best practices, the Prebid SDK should be initialized inside the adapter. But such changes may require deeper analysis to avoid issues with possible repetitive initializations.

So in this task's scope, we will introduce the fix, and the flow improvement will be performed eventually.